feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/ui/map/RouteView.kt

```diff
@@ -12,23 +15,32 @@ import com.epmedu.animeal.router.model.RouteResult
 import com.epmedu.animeal.router.presentation.FeedingRouteState
 import com.epmedu.animeal.timer.data.model.TimerState
 import com.mapbox.maps.MapView
+import com.mapbox.maps.plugin.locationcomponent.location
 import com.mapbox.navigation.base.options.NavigationOptions
 import com.mapbox.navigation.base.route.NavigationRoute
 import com.mapbox.navigation.core.MapboxNavigation
 import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
+import com.mapbox.navigation.ui.maps.route.line.model.RouteLineResources
 
 @Composable
 internal fun RouteView(
     mapView: MapView,
     state: HomeState,
     onRouteResult: (result: RouteResult) -> Unit
 ) {
-    val mapBoxRouteInitOptions = rememberMapRouteInitOptions(
-        mapView = mapView,
-        mapBoxNavigationInitOptions = MapBoxRouteInitOptions(
-            MapboxRouteLineOptions.Builder(mapView.context).build()
-        )
-    )
+    mapView.setMapViewLocationPuck(mapView.context)
```

Comment:
It doesn't make much sense to pass context, since you can retrieve it from MapView inside the function.

Reply:
that is correct, I had an incomplete refactor